templates/data-sources/provisioner.md.tmpl

```diff
@@ -0,0 +1,28 @@
+---
+Title: "Provisioner data source in a management cluster"
+Description: |-
+  Provisioner data source in a management cluster.
+---
+
+# Management Cluster Provisioner
+
+Read provisioner in a management cluster using this Terraform module.
+
+### Tanzu Kubernetes Grid Service and Tanzu Kubernetes Grid vSphere
+- Only provisioner read feature is enabled for Tanzu Kubernetes Grid Service and Tanzu Kubernetes Grid vSphere management clusters.
+- For a Tanzu Kubernetes cluster running in vSphere, you add a provisioner by creating
```

Comment:
Like above comment, creation is anyway covered in the resources mayn't be needed in datasource docs